Cesium.CodeGen.Tests/CodeGenContinueStatementTests.cs

```diff
@@ -48,4 +48,31 @@ public Task ContinueInDoWhile() => DoTest(@"int main()
 {
     continue;
 }"));
+
+    [Fact]
+    public void ContinueInSwitch() => DoesNotCompile(@"int main()
+{
+    switch (1)
+    {
+    default:
+        continue;
+    }
+}", "Can't break not from loop statement");
```

Comment:
_(Minor)_ Hm, the error message here may be clearer: you actually _can_ `break` from a non-loop construct, are you? It's only impossible to `continue` outside of a loop.

Let's consider rephrasing as "Can't use continue outside of a loop construct."